js/scripts.js

```diff
@@ -597,6 +596,7 @@ var plugin_formcreator = new function() {
          plugin_formcreator.dirty = false;
       }).done(function(response) {
          plugin_formcreator.dirty = false;
+         that.resetTabs();
```

Comment:
when qhestions are moved, reset other tabs of the form to force reload of preview when the admin wants to test the form